deploy/openshift/template.yaml

```diff
@@ -0,0 +1,149 @@
+          protocol: TCP
+          appProtocol: http
+          port: '${{SERVICE_PORT}}'
+          targetPort: http
+      selector:
+        app: '${APP_NAME}'
+  - kind: Deployment
+    apiVersion: apps/v1
+    metadata:
+      name: '${APP_NAME}'
+    spec:
+      replicas: '${{REPLICAS}}'
+      selector:
+        matchLabels:
+          app: '${APP_NAME}'
+      template:
+        metadata:
+          labels:
+            app: '${APP_NAME}'
+        spec:
+          affinity:
+            podAntiAffinity:
+              preferredDuringSchedulingIgnoredDuringExecution:
+                - podAffinityTerm:
+                    labelSelector:
+                      matchExpressions:
+                      - key: app
+                        operator: In
+                        values:
+                        - '${APP_NAME}'
+                    topologyKey: kubernetes.io/hostname
+                  weight: 90
+                - podAffinityTerm:
+                    labelSelector:
+                      matchExpressions:
+                      - key: app
+                        operator: In
+                        values:
+                        - '${APP_NAME}'
+                    topologyKey: topology.kubernetes.io/zone
+                  weight: 100
+          containers:
+            - name: app
+              image: '${IMAGE}:${IMAGE_TAG}'
+              livenessProbe:
+                httpGet:
+                  path: /q/health/live
+                  port: '${{SERVICE_PORT}}'
+                initialDelaySeconds: 15
+                periodSeconds: 20
+              readinessProbe:
+                httpGet:
+                  path: /q/health/ready
+                  port: '${{SERVICE_PORT}}'
+                initialDelaySeconds: 5
+                periodSeconds: 10
+              ports:
+                - name: http
+                  containerPort: '${{SERVICE_PORT}}'
+                  protocol: TCP
+              env:
+                - name: API_SNYK_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      name: exhort-secret
+                      key: api-snyk-token
+              securityContext:
+                runAsNonRoot: true
+              resources:
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+              imagePullPolicy: Always
+          restartPolicy: Always
+
+parameters:
+  - name: APP_NAME
+    displayName: Application name
+    description: Application name
+    value: exhort
+    required: true
+  - name: REPLICAS
+    displayName: Replicas
+    description: Number of desired pods
+    value: '1'
+    required: true
+  - name: IMAGE
+    displayName: Container image name
+    description: Container image name
+    value: quay.io/redhat-user-workloads/trusted-content-tenant/exhort-alpha
```

Comment:
There should be `quay.io/redhat-user-workloads/trusted-content-tenant/exhort-alpha/exhort`

Maybe on the first iteration, we can use `quay.io/ecosystem-appeng/exhort` before we understand how to customize the RHTAP pipeline